src/api/Feed.js

```diff
@@ -24,6 +25,7 @@ import {
     VERSION_UPLOAD_ACTION,
     VERSION_RESTORE_ACTION,
     TYPED_ID_FEED_PREFIX,
+    PLACEHOLDER_USER,
```

Comment:
One thing I am curious, what are the use cases when we have to fill in placeholder users? I notice that we are inserting user id with 2, which is reserved for anonymous users. But we don't show feed for anonymous users.

Instead of putting in placeholder users, is it better to just do null checks in components with the assumption that the user object may be absent? Or is some special UI being generated for placeholder user?

Reply:
@JustinHoldstock can correct me if I'm wrong but I heard that it's for the use case where a user has been deleted from the system, in which case the API will return null for the user. 

I think at the end of the day, the components should probably have null checks knowing that users could come in as null. This should fix the issue for ContentSidebar but say another ui-element ends up using comment (annotations?) they'd have to make a similar fix or make the fix in Comment itself. We can look at handling this in the annotations react refactoring

Reply:
If thats the case I would say we should just go ahead and do the simpler null checks / defensive approach for the components. Each component should know what to do with a potential null value. I also know that there is at least one use case where we use that null check to not show something. So if we add a placeholder, that will result in an incorrect experience, specially if the placeholder is actually a possible value.

Reply:
Ok let me close this PR then and do a new one with handling just in the components affected by this issue in sidebar